docs/data-sources/vault_cluster.md

```diff
@@ -81,6 +81,16 @@ Read-Only:
 - `elasticsearch_user` (String) ElasticSearch user for streaming audit logs
 - `grafana_endpoint` (String) Grafana endpoint for streaming audit logs
 - `grafana_user` (String) Grafana user for streaming audit logs
+- `http_basic_password` (String) HTTP basic authentication password for streaming audit logs, one of the two available authentication methods, can be specified only if http_bearer_token is not provided
```

Comment:
I think instead of `can be specified only if http_bearer_token is not provided` we can mention that it should `can be specified only if http_basic_user is also provided`